plugins/outputs/zabbix/lld_test.go

```diff
@@ -0,0 +1,797 @@
+package zabbix
+
+import (
+	"os"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/config"
+	"github.com/influxdata/telegraf/testutil"
+)
+
+type (
+	Operations                      interface{}
+	OperationAdd                    []telegraf.Metric
+	OperationPush                   struct{}
+	OperationCheck                  []telegraf.Metric
+	OperationCrossClearIntervalTime struct{}
```

Comment:
Why do you need this instead of directly using those types?

Reply:
I see. Can you please add this comment above these definitions!?

Reply:
fixed in 345cc538f